libunwind/src/AddressSpace.hpp

```diff
@@ -238,11 +238,6 @@ class _LIBUNWIND_HIDDEN LocalAddressSpace {
     return get<v128>(addr);
   }
   capability_t     getCapability(pint_t addr) { return get<capability_t>(addr); }
-  addr_t           getAddr(pint_t addr) {
-    // FIXME: for CHERI the actually address is the second 8 byte sequence
-    return get<addr_t>(addr);
-  }
-  __attribute__((always_inline))
   uintptr_t       getP(pint_t addr);
```

Comment:
Why did you remove the always_inline?

Reply:
Misparsed this as belonging to get<addr_t>, I will put it back.